code/rachel/HTML/lab01.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Salem_the_Bio</title>
+        <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+        <link rel="stylesheet" href="/Users/rcleav/Desktop/PDX_projects/class_swordfish/code/rachel/HTML/lab01_style.css"> <!--tried using relative path here - didn't work-->
```

Comment:
* Need to use a relative path here. It's important to use relative paths so the code works everywhere. See me or Merritt if you want help figuring out how to set it.